### ISSUE TITLE
Remove HEALTH.HealthCareFacilities from SGID

### ISSUE THREAD

**Issue by ZachBeck** [agrc/porter#256]
# Summary

- **Internal Dataset Name**: SGID.HEALTH.HealthCareFacilities
- **External Dataset Name**: opensgid.health.health_care_facilities
- **ArcGIS Online Url**: https://utah.maps.arcgis.com/home/item.html?id=f5e5d7c717c946fe9cd6f2c8d0bf8d86

HEALTH.HealthCareFacilities hasn't been updated in years. To facilitate the update process the current HealthCareFacilities needs to be deprecated and replaced by a dataset with the same name and new schema that is the same as the source data from the Dept of Health. 

## Migration Guide

AGOL data will be deprecated and shelved to make way for the new health care facilities

This dataset will be replaced by HEALTH.HealthCareFacilities which is named health.health_care_facilities in the Open SGID.
The replaced data is still accessible via our shelved policy in AGOL (a link to the shelved item).

## Action items

1. _Assign a person who should complete the task by replacing `name` with their github `@name`._
1. _Check [x] the box when the task is completed and add the date of completion._
1. _~Strike~ out all items that do not apply._

### Soft Delete

_The purpose of the soft delete is to ensure that all of our users and applications have gracefully migrated off of the dataset. Soft deletes will remain in effect for 14 days. During this time, we will have the ability to restore the dataset to its original SGID offering(s). After these 14 days, the item is then ready for a hard delete._

Note: If this dataset is being replaced, then wait until the new data is publicly available before completing these steps:

- [x] Append "(Mature Support)" to the end of the item title in the `SGID.META.AGOLItems` table (@ZachBeck , completed: `2024/07/08`)
- [x] Remove all tags other than "Deprecated" in the SDE metadata (@ZachBeck , completed: `2024/07/08`)
- [x] Add note to SDE description noting when layer will be deleted and any replacement layer (@ZachBeck, completed: `2024/07/08`)
- [x] Update the [SGID Index](https://docs.google.com/spreadsheets/d/11ASS7LnxgpnD0jN4utzklREgMf1pcvYjcXcIcESHweQ/edit#gid=1) row as deprecated by pasting the related Porter issue URL in the Deprecated field (@ZachBeck, completed: `2024/07/08`)
- [x] Change `Authoritative` field to `d` in `SGID.META.AGOLItems` to automatically set the `Deprecated` AGOL flag. Allow the `d` to persist through one run of [Auditor](https://github.com/agrc/auditor) - currently, Auditor runs daily at 5:00am (@ZachBeck , completed: `2024/07/08`)
  - After at least one successful run of Auditor:
    - [x] Remove the row from the `SGID.META.AGOLItems` table. This will trigger the removal of this item in Open SGID (@ZachBeck, completed: `2024/07/15`)
    - [x] Unshare the item from its SGID group, which will remove it from Open Data (@ZachBeck , completed: `2024/09/17`)

### Hard Delete

**Target Date:** 2024/07/23

_Hard deletes are final. It is recommended to complete the soft delete process before moving on to these steps. If you decide to skip the soft delete, note that you will need to incorporate some of those steps here._

- [x] Manually remove data from the Internal SGID (@ZachBeck , completed: `2024/07/23`)
  - [x] Deprecated database layers should be backed up on Google Drive > AGRC Projects > SGID > deprecated layers > [internal_db_sgid_layers](https://drive.google.com/drive/u/0/folders/10Fk8NI2UpEUnAbgvhjlN18pyvSFDWyIq) (@ZachBeck , completed: `2024/07/22`).
- [x] Remove ArcGIS Online item: (@ZachBeck  , completed: `2024/09/17`)
  - Manually delete the AGOL item
  - -OR-
  - Unshare by changing the `AGOL_ITEM_ID` field in `SGID.META.AGOLItems` to something other than an Item ID and manaually changing the sharing settings (remove the item from all SGID groups, remove Public sharing),
  - -OR-
  - Shelve the data by copying the row from `SGID.META.AGOLItems` to the `AGOLItems_shelved` table in AGOL and changing the `AGOL_ITEM_ID` field in `SGID.META.AGOLItems` to `shelved` or some other note.
  ~- Deprecated AGOL items can be backed up on Google Drive > AGRC Projects > SGID > deprecated layers >[agol_sgid_layers](https://drive.google.com/drive/u/0/folders/1xwSxiDNIH-9Hhmn6I7NfKBhm4SbYonX5).~
- ~[ ] Remove Farm from AGOL connection (name, completed: `2023/00/00`)~
- [x] Update relevant [gis.utah.gov](https://gis.utah.gov/data) data pages (@ZachBeck, completed: `2023/07/23`)
~- [ ] Add this porter url to the `Deprecated` field of the [Stewardship](https://docs.google.com/spreadsheets/d/11ASS7LnxgpnD0jN4utzklREgMf1pcvYjcXcIcESHweQ/edit#gid=1) record (name, completed: `2023/00/00`)~
- [x] Update `SGID.META.AGOLItems` table (@ZachBeck, completed: `2024/07/15`)
  - cut and paste row to `AGOLItems_shelved` table if shelving (see below)
  - set the `AGOL_ITEM_ID` field to `hosted by <agency>` for Farm from AGOL
  - set the `AGOL_ITEM_ID` field to `exclude from AGOL` to not publish to ArcGIS Online
- [x] Delete Google Drive data (@ZachBeck , completed: `2024/07/23`)
- [x] Remove row from `SGID.META.ChangeDetection` (@stdavis , completed: `2023/07/29`)
- [x] Remove data from forklift hashing and receiving (@stdavis, completed: `2023/07/29`)
- [x] Remove row from `data/hashed/changedetection.gdb/TableHashes` (@stdavis, completed: `2023/07/29`)

### Shelve/Static

_Choose one based on situation._

~- [ ] Upload to `UtahAGRC/AGRC_Shelved` folder in AGOL (New shelved item not already in AGOL) (name , completed: `2023/00/00`)~
~- [x] Move existing AGOL item to `AGRC_Shelved` AGOL folder (shelving an item already in AGOL) (@ZachBeck, completed: `2024/07/23`)~
~- [ ] Upload to appropriate `UtahAGRC/{SGID Category}` folder in AGOL (for `static` datasets) (name, completed: `2023/00/00`)~

_Add record to table._

~- [x] Add record to `AGOLItems_shelved` [table](https://utah.maps.arcgis.com/home/item.html?id=1760fbedbc7e49429aa6c0c3ab1442ec) in ArcGIS Online  with `shelved` or `static` in the `CATEGORY` field (@ZachBeck, completed: `2023/07/23`)~

## :robot: Automation validation

1. _Assign yourself or someone to check the item by replacing `name` with their github `@name`._
1. _Check [x] the box and add the date of verification `2020/01/01` when the task is verified._
1. _~Strike~ out all items that do not apply._

- [x] Remove data from Open SGID (@steveoh on `2024/08/07`)
- [x] [sgid-index](https://gis.utah.gov/data/sgid-index) (@steveoh on `2024/08/07`)
- [x] Auditor sets appropriate `shelved`/`static`/`Deprecated` information (@jacobdadams on `2024/07/09`)

## Are there service dependencies

- [x] Are clients querying this data with the WebAPI?
- [x] Notify Don Jackson (at motorolasolutions.com) if the data is in the Next-Generation 911 Motorola Aware Map? (@gregbunce on `2023/09/17`)
- [x] Does any other application depend on this data?

## Notification

- [x] Twitter (@steveoh)

## Group Task Assignments

1. _Check [x] the box when you have assigned all the tasks relevant to your group._

- [x] Data Team (@gregbunce)
- [x] Dev Team (@steveoh)
- [x] Cadastre Team (@rkelson)

**Comment by agrc-conductor** [comment]
## conductor results for tasks - 256

| check | status |
| - | :-: |
| @ZachBeck has completed **0** out of **11** tasks | :no_entry: |
| @jacobdadams has completed **0** out of **4** tasks | :no_entry: |
| @stdavis has completed **0** out of **3** tasks | :no_entry: |
| @steveoh has completed **0** out of **3** tasks | :no_entry: |
| @eneemann has completed **0** out of **1** tasks | :no_entry: |
| @gregbunce has completed **0** out of **1** tasks | :no_entry: |
| @rkelson has completed **0** out of **1** tasks | :no_entry: |

**Comment by agrc-conductor** [comment]
## conductor results for tasks - 256

| check | status |
| - | :-: |
| @ZachBeck has completed **0** out of **11** tasks | :no_entry: |
| @jacobdadams has completed **0** out of **4** tasks | :no_entry: |
| @stdavis has completed **0** out of **3** tasks | :no_entry: |
| @steveoh has completed **0** out of **3** tasks | :no_entry: |
| @eneemann has completed **0** out of **1** tasks | :no_entry: |
| @gregbunce has completed **0** out of **1** tasks | :no_entry: |
| @rkelson has completed **0** out of **1** tasks | :no_entry: |

**Comment by agrc-conductor** [comment]
## conductor results for tasks - 256

| check | status |
| - | :-: |
| @ZachBeck has completed **0** out of **11** tasks | :no_entry: |
| @jacobdadams has completed **0** out of **4** tasks | :no_entry: |
| @stdavis has completed **0** out of **3** tasks | :no_entry: |
| @steveoh has completed **0** out of **3** tasks | :no_entry: |
| @eneemann has completed **0** out of **1** tasks | :no_entry: |
| @gregbunce has completed **0** out of **1** tasks | :no_entry: |
| @rkelson has completed **0** out of **1** tasks | :no_entry: |

**Comment by agrc-conductor** [comment]
## conductor results for tasks - 256

| check | status |
| - | :-: |
| @ZachBeck has completed **0** out of **11** tasks | :no_entry: |
| @jacobdadams has completed **0** out of **4** tasks | :no_entry: |
| @stdavis has completed **0** out of **3** tasks | :no_entry: |
| @steveoh has completed **0** out of **3** tasks | :no_entry: |
| @eneemann has completed **0** out of **1** tasks | :no_entry: |
| @gregbunce has completed **0** out of **1** tasks | :no_entry: |
| @rkelson has completed **0** out of **1** tasks | :no_entry: |

**Comment by agrc-conductor** [comment]
## conductor results for tasks - 256

| check | status |
| - | :-: |
| @ZachBeck has completed **0** out of **11** tasks | :no_entry: |
| @jacobdadams has completed **0** out of **4** tasks | :no_entry: |
| @stdavis has completed **0** out of **3** tasks | :no_entry: |
| @steveoh has completed **0** out of **3** tasks | :no_entry: |
| @gregbunce has completed **0** out of **2** tasks | :no_entry: |
| @rkelson has completed **0** out of **1** tasks | :no_entry: |

**Comment by steveoh** [comment]
adding blocked since work hasn't started

**Comment by ZachBeck** [comment]
Waiting on updated facilities from Health.

**Comment by agrc-conductor** [comment]
## conductor results for HEALTH.HealthCareFacilities

| check | status |
| - | :-: |
| internal sgid | :no_entry: |
| meta table | :no_entry: |
| open sgid | :no_entry: |
| open data | :no_entry: |
| arcgis online | :no_entry: |
| stewardship |  |
| - porter issue link | :no_entry: |

**Comment by agrc-conductor** [comment]
## conductor results for tasks - 256

| check | status |
| - | :-: |
| @ZachBeck has completed **0** out of **11** tasks | :no_entry: |
| @jacobdadams has completed **0** out of **4** tasks | :no_entry: |
| @stdavis has completed **0** out of **3** tasks | :no_entry: |
| @steveoh has completed **0** out of **3** tasks | :no_entry: |
| @gregbunce has completed **1** out of **2** tasks | :no_entry: |
| rkelson has completed **1** out of **1** tasks | :+1: |

**Comment by agrc-conductor** [comment]
## conductor results for HEALTH.HealthCareFacilities

| check | status |
| - | :-: |
| internal sgid | :no_entry: |
| meta table | :no_entry: |
| open sgid | :no_entry: |
| open data | :no_entry: |
| arcgis online | :no_entry: |
| stewardship |  |
| - porter issue link | :no_entry: |

**Comment by agrc-conductor** [comment]
## conductor results for tasks - 256

| check | status |
| - | :-: |
| @ZachBeck has completed **0** out of **11** tasks | :no_entry: |
| @jacobdadams has completed **0** out of **4** tasks | :no_entry: |
| @stdavis has completed **0** out of **3** tasks | :no_entry: |
| @steveoh has completed **0** out of **3** tasks | :no_entry: |
| @gregbunce has completed **1** out of **2** tasks | :no_entry: |
| rkelson has completed **1** out of **1** tasks | :+1: |

**Comment by agrc-conductor** [comment]
## conductor results for HEALTH.HealthCareFacilities

| check | status |
| - | :-: |
| internal sgid | :no_entry: |
| meta table | :no_entry: |
| open sgid | :no_entry: |
| open data | :no_entry: |
| arcgis online | :no_entry: |
| stewardship |  |
| - porter issue link | :no_entry: |

**Comment by agrc-conductor** [comment]
## conductor results for tasks - 256

| check | status |
| - | :-: |
| @ZachBeck has completed **0** out of **11** tasks | :no_entry: |
| @jacobdadams has completed **0** out of **4** tasks | :no_entry: |
| @stdavis has completed **0** out of **3** tasks | :no_entry: |
| @steveoh has completed **0** out of **3** tasks | :no_entry: |
| @gregbunce has completed **1** out of **2** tasks | :no_entry: |
| rkelson has completed **1** out of **1** tasks | :+1: |

**Comment by brigsz** [comment]
Do folks think this will need a new retention schedule as it moves to "Shelved"? it seems like it will, as it is different from non shelved.

**Comment by steveoh** [comment]
In my opinion, it lived in the SGID for longer than 6 months and archives should have a copy so I believe we are covered there. The schedule it is on says archives will retain it in perpetuity, which I would suggest maybe a shorter lifespan since it doesn't appear to have historical relevance.

**Comment by agrc-conductor** [comment]
## conductor results for HEALTH.HealthCareFacilities

| check | status |
| - | :-: |
| internal sgid | :no_entry: |
| meta table | :no_entry: |
| open sgid | :no_entry: |
| open data | :no_entry: |
| arcgis online | :no_entry: |
| stewardship |  |
| - porter issue link | :no_entry: |

**Comment by agrc-conductor** [comment]
## conductor results for tasks - 256

| check | status |
| - | :-: |
| @ZachBeck has completed **0** out of **11** tasks | :no_entry: |
| @jacobdadams has completed **0** out of **4** tasks | :no_entry: |
| @stdavis has completed **0** out of **3** tasks | :no_entry: |
| @steveoh has completed **0** out of **3** tasks | :no_entry: |
| @gregbunce has completed **1** out of **2** tasks | :no_entry: |
| rkelson has completed **1** out of **1** tasks | :+1: |

**Comment by agrc-conductor** [comment]
## conductor results for HEALTH.HealthCareFacilities

| check | status |
| - | :-: |
| internal sgid | :no_entry: |
| meta table | :no_entry: |
| open sgid | :+1: |
| open data | :no_entry: |
| arcgis online | :no_entry: |
| stewardship |  |
| - porter issue link | :no_entry: |

**Comment by agrc-conductor** [comment]
## conductor results for tasks - 256

| check | status |
| - | :-: |
| @ZachBeck has completed **4** out of **13** tasks | :no_entry: |
| @jacobdadams has completed **1** out of **4** tasks | :no_entry: |
| @stdavis has completed **0** out of **3** tasks | :no_entry: |
| @steveoh has completed **0** out of **3** tasks | :no_entry: |
| @gregbunce has completed **1** out of **2** tasks | :no_entry: |
| rkelson has completed **1** out of **1** tasks | :+1: |

**Comment by agrc-conductor** [comment]
## conductor results for HEALTH.HealthCareFacilities

| check | status |
| - | :-: |
| internal sgid | :no_entry: |
| meta table | :no_entry: |
| open sgid | :no_entry: |
| open data | :no_entry: |
| arcgis online | :no_entry: |
| stewardship |  |
| - porter issue link | :+1: |

**Comment by agrc-conductor** [comment]
## conductor results for tasks - 256

| check | status |
| - | :-: |
| @ZachBeck has completed **6** out of **13** tasks | :no_entry: |
| @jacobdadams has completed **1** out of **4** tasks | :no_entry: |
| @stdavis has completed **0** out of **3** tasks | :no_entry: |
| @steveoh has completed **0** out of **3** tasks | :no_entry: |
| @gregbunce has completed **1** out of **2** tasks | :no_entry: |
| rkelson has completed **1** out of **1** tasks | :+1: |

**Comment by jacobdadams** [comment]
Sorry to reopen the conversation. We're shelving this layer, but I'm wondering if we shouldn't just do a hard delete it and keep a copy in our google drive. It seems as though it's been completely superseded by the new layer and there's not any reason to keep the historic data around (ie, is anyone going to want to compare this with the new one to track changes over time?).

**Comment by steveoh** [comment]
I agree. When Matt and I were looking at the shelved category items, it appeared that there are many more shelved items than seemed to have historical relevance. Maybe I need a refresher on when items should be shelved or we need to take a closer look at the process to understand what's being added there.

**Comment by jacobdadams** [comment]
From our policy:
>Over time, we will remove datasets from the Open SGID that are no longer current but continue to have historical value (e.g., Census/ACS data or previous tax district boundaries). Their associated ArcGIS Online items will be tagged as shelved and placed in the public UGRC Shelf ArcGIS Online group for future reference. Shelved datasets are not included in the SGID on ArcGIS site.

**Comment by steveoh** [comment]
Tracking in https://github.com/agrc/planning-queue/issues/452

**Comment by agrc-conductor** [comment]
## conductor results for tasks - 256

| check | status |
| - | :-: |
| @ZachBeck has completed **11** out of **13** tasks | :no_entry: |
| @jacobdadams has completed **1** out of **4** tasks | :no_entry: |
| @stdavis has completed **0** out of **3** tasks | :no_entry: |
| @steveoh has completed **0** out of **3** tasks | :no_entry: |
| @gregbunce has completed **1** out of **2** tasks | :no_entry: |
| rkelson has completed **1** out of **1** tasks | :+1: |

**Comment by agrc-conductor** [comment]
## conductor results for HEALTH.HealthCareFacilities

| check | status |
| - | :-: |
| internal sgid | :+1: |
| meta table | :+1: |
| stewardship |  |
| - porter issue link | :+1: |

**Comment by agrc-conductor** [comment]
## conductor results for tasks - 256

| check | status |
| - | :-: |
| @ZachBeck has completed **11** out of **13** tasks | :no_entry: |
| @jacobdadams has completed **1** out of **4** tasks | :no_entry: |
| stdavis has completed **3** out of **3** tasks | :+1: |
| @steveoh has completed **0** out of **3** tasks | :no_entry: |
| @gregbunce has completed **1** out of **2** tasks | :no_entry: |
| rkelson has completed **1** out of **1** tasks | :+1: |

**Comment by agrc-conductor** [comment]
## conductor results for HEALTH.HealthCareFacilities

| check | status |
| - | :-: |
| internal sgid | :+1: |
| meta table | :+1: |
| stewardship |  |
| - porter issue link | :+1: |

**Comment by agrc-conductor** [comment]
## conductor results for tasks - 256

| check | status |
| - | :-: |
| @ZachBeck has completed **11** out of **13** tasks | :no_entry: |
| @jacobdadams has completed **1** out of **3** tasks | :no_entry: |
| stdavis has completed **3** out of **3** tasks | :+1: |
| @steveoh has completed **3** out of **4** tasks | :no_entry: |
| @gregbunce has completed **1** out of **2** tasks | :no_entry: |
| rkelson has completed **1** out of **1** tasks | :+1: |

**Comment by agrc-conductor** [comment]
## conductor results for HEALTH.HealthCareFacilities

| check | status |
| - | :-: |
| internal sgid | :+1: |
| meta table | :+1: |
| stewardship |  |
| - porter issue link | :+1: |

**Comment by agrc-conductor** [comment]
## conductor results for HEALTH.HealthCareFacilities

| check | status |
| - | :-: |
| internal sgid | :+1: |
| meta table | :+1: |
| stewardship |  |
| - porter issue link | :+1: |

**Comment by agrc-conductor** [comment]
## conductor results for tasks - 256

| check | status |
| - | :-: |
| @ZachBeck has completed **11** out of **13** tasks | :no_entry: |
| @jacobdadams has completed **1** out of **3** tasks | :no_entry: |
| stdavis has completed **3** out of **3** tasks | :+1: |
| steveoh has completed **4** out of **4** tasks | :+1: |
| @gregbunce has completed **1** out of **2** tasks | :no_entry: |
| rkelson has completed **1** out of **1** tasks | :+1: |

**Comment by agrc-conductor** [comment]
## conductor results for HEALTH.HealthCareFacilities

| check | status |
| - | :-: |
| internal sgid | :+1: |
| meta table | :+1: |
| stewardship |  |
| - porter issue link | :+1: |

**Comment by agrc-conductor** [comment]
## conductor results for tasks - 256

| check | status |
| - | :-: |
| ZachBeck has completed **13** out of **13** tasks | :+1: |
| @jacobdadams has completed **1** out of **3** tasks | :no_entry: |
| stdavis has completed **3** out of **3** tasks | :+1: |
| steveoh has completed **4** out of **4** tasks | :+1: |
| @gregbunce has completed **1** out of **2** tasks | :no_entry: |
| rkelson has completed **1** out of **1** tasks | :+1: |

**Comment by agrc-conductor** [comment]
## conductor results for HEALTH.HealthCareFacilities

| check | status |
| - | :-: |
| internal sgid | :+1: |
| meta table | :+1: |
| stewardship |  |
| - porter issue link | :+1: |

**Comment by agrc-conductor** [comment]
## conductor results for tasks - 256

| check | status |
| - | :-: |
| ZachBeck has completed **13** out of **13** tasks | :+1: |
| @jacobdadams has completed **1** out of **3** tasks | :no_entry: |
| stdavis has completed **3** out of **3** tasks | :+1: |
| steveoh has completed **4** out of **4** tasks | :+1: |
| @gregbunce has completed **1** out of **2** tasks | :no_entry: |
| rkelson has completed **1** out of **1** tasks | :+1: |

**Comment by agrc-conductor** [comment]
## conductor results for HEALTH.HealthCareFacilities

| check | status |
| - | :-: |
| internal sgid | :+1: |
| meta table | :+1: |
| stewardship |  |
| - porter issue link | :+1: |

**Comment by agrc-conductor** [comment]
## conductor results for tasks - 256

| check | status |
| - | :-: |
| ZachBeck has completed **13** out of **13** tasks | :+1: |
| @jacobdadams has completed **1** out of **3** tasks | :no_entry: |
| stdavis has completed **3** out of **3** tasks | :+1: |
| steveoh has completed **4** out of **4** tasks | :+1: |
| @gregbunce has completed **1** out of **2** tasks | :no_entry: |
| rkelson has completed **1** out of **1** tasks | :+1: |

**Comment by agrc-conductor** [comment]
## conductor results for HEALTH.HealthCareFacilities

| check | status |
| - | :-: |
| internal sgid | :+1: |
| meta table | :+1: |
| stewardship |  |
| - porter issue link | :+1: |

**Comment by agrc-conductor** [comment]
## conductor results for tasks - 256

| check | status |
| - | :-: |
| ZachBeck has completed **13** out of **13** tasks | :+1: |
| @jacobdadams has completed **1** out of **3** tasks | :no_entry: |
| stdavis has completed **3** out of **3** tasks | :+1: |
| steveoh has completed **4** out of **4** tasks | :+1: |
| @gregbunce has completed **1** out of **2** tasks | :no_entry: |
| rkelson has completed **1** out of **1** tasks | :+1: |

**Comment by agrc-conductor** [comment]
## conductor results for HEALTH.HealthCareFacilities

| check | status |
| - | :-: |
| internal sgid | :+1: |
| meta table | :+1: |
| stewardship |  |
| - porter issue link | :+1: |

**Comment by ZachBeck** [comment]
done
